spec/message.ebnf

```diff
@@ -1,25 +1,20 @@
-Message ::= Declaration* ( Pattern | Selector Variant+ )
+Message ::= (s? Declaration)* s? Body s?
 
-/* Preamble */
-Declaration ::= 'let' WhiteSpace Variable '=' '{' Expression '}'
-Selector ::= 'match' ( '{' Expression '}' )+
+Declaration ::= 'let' s Variable s? '=' s? '{' s? Expression s? '}'
+Body ::= Pattern | Selector (s? Variant)+
```

Comment:
```suggestion
Body ::= Pattern | Selector (s Variant)+
```
Same as with `Message`, without a required space we would allow for
```
match {$foo}when 1{yes}when *{no}
```